server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -252,6 +252,7 @@ public void apply(Settings value, Settings current, Settings previous) {
                 Metadata.SETTING_READ_ONLY_SETTING,
                 Metadata.SETTING_READ_ONLY_ALLOW_DELETE_SETTING,
                 ShardLimitValidator.SETTING_CLUSTER_MAX_SHARDS_PER_NODE,
+                ShardLimitValidator.SETTING_CLUSTER_IGNORE_HIDDEN_INDEXES,
```

Comment:
indices beginning with `.` are not necessarily hidden and vice-versa. `index.hidden` is a separate index level setting. Many opensearch plugin use that like cross-cluster and many don't like security-plugin.   Should we say `dot_indices` ?

Reply:
Previously named this to dot_index however was asked to rename as mentioned in this close CR https://github.com/opensearch-project/OpenSearch/pull/4674

Reply:
@Bukhtawar : what is your take on using `hidden` or `dot` ?

Reply:
dot indices have been deprecated so going forward we should use hidden and system indices only. For 2.x we might still have to support dot indices. Any reason why security plugin is still using dot index?

Reply:
@Bukhtawar @gbbafna so should we change to dot or go with hidden ??